stix_shifter_modules/qradar/stix_translation/json/to_stix_map.json

```diff
@@ -459,6 +459,21 @@
       "object": "x-qradar"
     }
   ],
+  "dnsdomainname": [
+    {
+      "key": "url.value",
```

Comment:
The goal here was to get the URL/hostname information exposed so that something like Threat Intelligence Insights in CP4S could detect it when performing a scan based on an X-Force threat feed that have URLs/hostnames in their collection. 

Is there a more suitable way to achieve the same goal? If so, should we apply that same pattern to the `tlsservernameindication` and `httphost` fields that can have similarly formatted values?

Reply:
If I understood it correctly, you need url object for threat detection for all these fields, right? Ideally its not recommended or sometimes invalid according to STIX specification but for your use case I don't see any other mapping options. 
we usually ask people to create uniform custom object such as `x-qradar-url`. Not sure if that would help your case.